src/applications/simple-forms-form-engine/shared/tests/unit/utils/formConfig.unit.spec.jsx

```diff
@@ -29,20 +29,28 @@ describe('createFormConfig', () => {
   let transformSpy;
 
   beforeEach(() => {
-    const FakeComponent = ({ ombInfo }) => (
+    const FakeComponent = ({ introParagraph, ombInfo, whatToKnow = [] }) => (
       <div>
+        <p data-testid="intro-paragraph">introParagraph: {introParagraph}</p>
+        <ul data-testid="what-to-know">
+          {whatToKnow.map((bullet, index) => (
+            <li key={index}>{bullet}</li>
+          ))}
```

Comment:
`index` as `key` generally catches my eye as a potentially problematic, but I think it should be fine in this context. These are not dynamic in any way.